packages/marketplace-ui/src/components/DisburseFunds.tsx

```diff
@@ -130,6 +132,9 @@ export const DisburseFunds = ({
                   destinationWallet: new PublicKey(address),
                   includeRetrievalCurve,
                 });
+                if (closeEntangler && childEntangler?.publicKey) {
+                  cache?.delete(childEntangler.publicKey);
```

Comment:
Weird. Does our cache not handle things getting deleted on chain?

Reply:
It seemed like it was hitting a cache at some point, but might've been something else. Not needed anymore anyway since we're redirecting to docs for disbursing funds, so I'll delete that.